docs/user/language-bindings/javascript/plugins/index.md

```diff
@@ -0,0 +1,37 @@
+# Plugins
+
+The Glean JavaScript SDK accepts a plugin array as an initialization parameter.
+
+```js
+import Glean from "@mozilla/glean/webext"
+// This is not a real available plugin,
+// it is a hypothetical one for illustration purposes.
+import HypotheticalPlugin from "@mozilla/glean/plugins/hypothetical"
+
+Glean.initialize(
+  "my.fancy.modified.app",
+  uploadStatus,
+  {
+    plugins: [
+      new HypotheticalPlugin("with", "hypothetical", "arguments")
+    ]
+  }
+);
+```
+
+Plugins attach to specific internal events on the SDK and can modify its behavior.
```

Comment:
Can you list out the currently supported events?

Reply:
I though about that, but that is internal detail for now. Since we don't let people write their own plugins, what is the point in listing events?

Reply:
Ok, let's move on for now

Reply:
(FWIW, there is a bug to document events on the Glean.js developer docs. It's [Bug 1698089](https://bugzilla.mozilla.org/show_bug.cgi?id=1698089) :D)